client/my-sites/plan-features/index.jsx

```diff
@@ -468,6 +483,7 @@ export default connect(
 				const newPlan = isNew( plan ) && ! isPaid;
 				const currentPlan = sitePlan && sitePlan.product_slug;
 				const showMonthlyPrice = ! relatedMonthlyPlan && showMonthly;
+				const currentPlanMatch = !! ( sitePlan && ( getPlanClass( sitePlan.product_slug ) === getPlanClass( plan ) ) );
```

Comment:
I agree this one's a little dirty, and it feels fragile to rely on the CSS classes. Who knows how our CSS standards might shift one day, causing unexpected bugs 😉 

How about defining a utility function similar to `getMonthlyPlanByYearly` that would return the opposite plan? This sounds like a clean and testable solution, and I doubt we'll be introducing weekly plans anytime soon 😉